en/guide/vuex-store.md

```diff
@@ -189,12 +189,18 @@ export default {
 }
 ```
 
-<div class="Alert">Vous pouvez également avoir des modules en exportant une instance de store vous devrez les ajouter manuellement sur votre store.</div>
+<div class="Alert">
+
+Vous pouvez également avoir des modules en exportant une instance de store vous devrez les ajouter manuellement sur votre store.
```

Comment:
Il y a un problème de syntaxe dans cette phrase

Reply:
la phrase en anglais est celle-ci :
You can also have modules by exporting a store instance, you will have to add them manually on your store.

peut-être à traduire ainsi (je n'y ai pas touché à part l'ajout de retour à la ligne pour coller avec la version anglaise) :
Vous pouvez également avoir des modules en exportant une instance de store, vous devrez alors les ajouter manuellement sur votre store.

Reply:
modifié dans le commit https://github.com/vuejs-fr/nuxt/pull/41/commits/9ce2be1d2042760f321659ac9ac756dd4a4e367a